Bottom: e1efc0f3e7744ece5658ea9de4ae2e0621c8d195
Top:    d623ee01351810bb6e79a480afac19019b71b4bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 19:41:09 +0100

hid/gtk: Render a line along side grouped layers, showing their grouping.


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 432e57e..7ce5b13 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -570,6 +570,8 @@ ghid_layer_selector_new (void)
  *  If the user_id passed already exists in the layer selector, that layer
  *  will have its data overwritten with the new stuff.
  *
+ * XXX: Add feature to mark adjacent layers which belong to the same group
+ *
  *  \param [in] ls            The selector to be acted on
  *  \param [in] user_id       An ID used to identify the layer; will be passed to selection/visibility callbacks
  *  \param [in] name          The name of the layer; will be used on selector and menus
